framework/src/engine/index.ts

```diff
@@ -12,5 +12,6 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-export { Engine } from './engine';
-export { computeValidatorsHash, areDistinctHeadersContradicting, BFTParameters } from './bft';
+export * from './engine';
```

Comment:
we only expose what we need from the file, so better to only expose `Engine` class only as previously done